app/components/FFXIVResults/ShortagePredictor/Results.tsx

```
@@ -0,0 +1,276 @@
+      columnId: 'chart_button',
+      header: 'Last 24 Hours',
+      accessor: ({ row }) => (
+        <button
+          className="inline-flex items-center rounded-md bg-black px-2.5 py-2 text-sm font-medium text-white"
+          onClick={() =>
+            onRowPress({
+              title: row.item_name,
+              p: row.chart_p,
+              q: row.chart_q
+            })
+          }>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)